mtprotosender.py:228 #INFO     [2025-09-25 02:31:02,469] - telethon.network.mtprotosender - 0| Connecting to 149.154.167.51:443/TcpFull...
mtprotosender.py:279 #INFO     [2025-09-25 02:31:02,510] - telethon.network.mtprotosender - 0| Connection to 149.154.167.51:443/TcpFull complete!
handlers.py:18 #INFO     [2025-09-25 02:31:05,040] - root - 0| Handler called. NewMessage. type = <class 'telethon.tl.patched.Message'>
api_interface.py:141 #INFO     [2025-09-25 02:31:05,514] - venv - 0| 200 | https://tenerife.chat/chat/webhook/telegram/create | {'success': True, 'method': 'create'}
handlers.py:18 #INFO     [2025-09-25 02:31:13,820] - root - 0| Handler called. NewMessage. type = <class 'telethon.tl.patched.Message'>
api_interface.py:141 #INFO     [2025-09-25 02:31:14,134] - venv - 0| 200 | https://tenerife.chat/chat/webhook/telegram/create | {'success': True, 'method': 'create'}
handlers.py:18 #INFO     [2025-09-25 02:31:21,849] - root - 0| Handler called. NewMessage. type = <class 'telethon.tl.patched.Message'>
api_interface.py:141 #INFO     [2025-09-25 02:31:22,045] - venv - 0| 200 | https://tenerife.chat/chat/webhook/telegram/create | {'success': True, 'method': 'create'}
handlers.py:18 #INFO     [2025-09-25 02:31:30,807] - root - 0| Handler called. NewMessage. type = <class 'telethon.tl.patched.Message'>
api_interface.py:141 #INFO     [2025-09-25 02:31:31,077] - venv - 0| 200 | https://tenerife.chat/chat/webhook/telegram/create | {'success': True, 'method': 'create'}
handlers.py:18 #INFO     [2025-09-25 02:31:40,463] - root - 0| Handler called. NewMessage. type = <class 'telethon.tl.patched.Message'>
api_interface.py:141 #INFO     [2025-09-25 02:31:40,634] - venv - 0| 200 | https://tenerife.chat/chat/webhook/telegram/create | {'success': True, 'method': 'create'}
mtprotosender.py:319 #INFO     [2025-09-25 02:31:46,808] - telethon.network.mtprotosender - 0| Disconnecting from 149.154.167.51:443/TcpFull...
mtprotosender.py:339 #INFO     [2025-09-25 02:31:46,808] - telethon.network.mtprotosender - 0| Disconnection from 149.154.167.51:443/TcpFull complete!
base_events.py:1758 #ERROR    [2025-09-25 02:31:46,847] - asyncio - 0| Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x00000247BFCEA7D0>
base_events.py:1758 #ERROR    [2025-09-25 02:31:46,848] - asyncio - 0| Unclosed connector
connections: ['deque([(<aiohttp.client_proto.ResponseHandler object at 0x00000247BFD0D240>, 103229.609)])']
connector: <aiohttp.connector.TCPConnector object at 0x00000247BFCEA740>
base_events.py:1758 #ERROR    [2025-09-25 02:31:46,848] - asyncio - 0| Fatal error on SSL transport
protocol: <asyncio.sslproto.SSLProtocol object at 0x00000247BF98E3B0>
transport: <_ProactorSocketTransport fd=840 read=<_OverlappedFuture cancelled>>
Traceback (most recent call last):
  File "C:\Users\Valentyn\AppData\Local\Programs\Python\Python310\lib\asyncio\sslproto.py", line 690, in _process_write_backlog
    self._transport.write(chunk)
  File "C:\Users\Valentyn\AppData\Local\Programs\Python\Python310\lib\asyncio\proactor_events.py", line 365, in write
    self._loop_writing(data=bytes(data))
  File "C:\Users\Valentyn\AppData\Local\Programs\Python\Python310\lib\asyncio\proactor_events.py", line 401, in _loop_writing
    self._write_fut = self._loop._proactor.send(self._sock, data)
AttributeError: 'NoneType' object has no attribute 'send'
